### DISCUSSION TITLE
How to enable Envoy's overload manager in Istio?

### DISCUSSION THREAD

**Discussion by namm2** [istio/istio#53364] [Q&A]
Hi, I'm using Istio ingress gateway and I'd like to enable Envoy's [Overload manager](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/operations/overload_manager). As far as I understand this is configured at bootstrap, but unfortunately Istio v1.21 removed the ability to patch bootstrap with EnvoyFilter. Then does anybody know how to achieve this task in Istio? Or is there any similar functionality of Envoy's overload manager in Istio that I can use? TYVM

**Comment by ramaraochavali** [comment]
https://github.com/istio/istio/issues/14366#issuecomment-548141596 - you can use this approach

**Reply by namm2** [reply]
> [#14366 (comment)](https://github.com/istio/istio/issues/14366#issuecomment-548141596) - you can use this approach

I'm aware of using ConfigMap, but as far as I can understand that way only works with the Istio sidecar which I don't use, instead my setup is just Istiod with Istio ingress gateway pods.

**Comment by ramaraochavali** [comment]
Even for Ingress Gateway it works.  You need to set ISTIO_BOOTSTRAP_OVERRIDE to the configmap location
